app/export.go

```diff
@@ -2,38 +2,49 @@ package app
 
 import (
 	"encoding/json"
+	"fmt"
```

Comment:
Can we use new logger right away for new files / logs ? No need to replace all the logs since it's been done in a separate PR, but for new logs it could be cleaner to use new approach initially

Reply:
im removed some fmt.Sprintf but for others its fine unless you are asking to concat strings??